api/api.yaml

```diff
@@ -16,130 +16,45 @@ servers:
         required: true
         content:
           application/json:
             schema:
-              $ref: '../config/config.yaml#/components/schemas/Config'
+              $ref: '../control/control.yaml#/components/schemas/Control.State'
       responses:
-        '204':
-          description: >-
-            The operation has successfully completed with no content.
         '202':
-          description: >-
-            The operation has been accepted and is in progress.
-            The response payload will contain two urls
-              - one to get the progress of the current operation
-                Each subsequent progress request can return one of the following:     
-                  - 204 if the operation has successfully 
-                  - 202 if the operation is still in progress
-                  - 400 if the operation has failed
-              - one to cancel the operation. 
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  progress_url:
-                    type: string
-                  cancellation_url:
-                    type: string
-        '400':
-          description: >-
-            The operation has failed.
-            The failure response will be a list errors.
-            The operation should not be repeated without modifications.
-          content:
-            application/json:
-              schema:
-                $ref: '../result/result.yaml#/components/schemas/Result.Errors'
-        '409':
-          description: >-
-            The request conflicts with an executing request.
-          content:
-            application/json:
-              schema:
-                $ref: '../result/result.yaml#/components/schemas/Result.Errors'
-          
-  /control/port/link:
-    description: >-
-      Control port link API
-    post:
-      tags:
-        - Control
-      operationId: set_port_link
-      description: >-
-        Sets the port link state on the traffic generator.
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '../control/control.yaml#/components/schemas/Control.PortLink'
-      responses:
-        '204':
-          description: >-
-            No content
-
-  /control/port/capture:
-    description: >-
-      Control port capture API
-    post:
-      tags:
-        - Control
-      operationId: set_port_capture
-      description: >-
-        Starts a port capture on the traffic generator.
-        
-        Use the /results/capture API to stop a port capture and get the captured
-        results.
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '../control/control.yaml#/components/schemas/Control.PortCapture'
-      responses:
+          $ref: '../result/request.yaml#/components/responses/Pending'
```

Comment:
are we removing 202 for now based on this morning's discussion to keep it synchronous?